README.md

```diff
@@ -132,3 +132,49 @@ For more information on the Akamai API, read the [property manager docs](https:/
 ## Build Process
 
 This repository has a webhook that automatically builds a [Jenkins job](https://jenkins-jenkins.5a9f.insights-dev.openshiftapps.com/job/akamai-config-deployer/) on every push. To configure this webhook, check the project's [webhook settings](https://github.com/RedHatInsights/cloud-services-config/settings/hooks)
+
+## Testing your changes locally
+
+Testing local changes is straightforward. First, add a line like this to your insights-proxy spandx config:
```

Comment:
Just a quick comment, following the [quick start guide](https://cloud.redhat.com/docs/storybook?path=/story/welcome--quick-start-guide) uses the file `config/spandx.config.js` instead of `profiles/local-frontend.js`.

I know that you are referring to: 
> your insights-proxy spandx config

but maybe it would be useful to use the defaults.

Note that I'm new here and maybe i'm wrong on assuming these are the defaults, this information was really helpful to me, thanks!